### ISSUE TITLE
В доступных пулах скрывать пулы которые ты полностью разметил

### ISSUE THREAD

**Issue by akmetainfo** [OpenCorpora/opencorpora#671]
Не сильно критично, но. Очень долго приходится проматывать все эти нолики, чтобы внизу увидеть один-два свежих пула и разметить по-быстрому.

![12344444](https://cloud.githubusercontent.com/assets/6085279/9038165/f87f4ff4-39fa-11e5-99c5-a24ce2be6af5.png)

**Comment by grandsbor** [comment]
Второй вариант проще

**Comment by grandsbor** [comment]
Т.е. юзкейс "просматривать свои старые ответы" тут не нужен?

**Comment by akmetainfo** [comment]
Верно. Я свои старые ответы смотрю на странице профиля. Ну, если мне вдруг такая идея в голову придёт: частично сделанные пулы остаются, в них можно зайти и поправить. А старые ответы, где пулы закрылись и ничего не поменять -- из профиля, где ачивки и ответы, намайненные артефакты ткзть.

**Comment by grandsbor** [comment]
cc @victorbocharov 
Есть предложение оторвать фичу "просмотр своих старых ответов" со страницы tasks.php
